src/providers/pubsub.ts

```diff
@@ -98,6 +98,12 @@ export class TopicBuilder {
   }
 }
 
+/**
+ * Registers a Cloud Function to run at specified times.
+ *
+ * @param schedule The scheduling logic, in Unix Crontab or AppEngine syntax.
```

Comment:
I would go with the "The schedule" rather than "The scheduler logic", since it's not really a logic. It's just a definition of the schedule.